src/components/dls/QuranWord/onClickQuranWord.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  triggerPauseAudio,
+  triggerPlayAudio,
+  triggerSetCurrentTime,
+} from 'src/components/AudioPlayer/EventTriggers';
+import { getVerseTimingByVerseKey, QURANCDN_AUDIO_BASE_URL } from 'src/utils/audio';
+import AudioData from 'types/AudioData';
+import Word from 'types/Word';
+
+/**
+ * Given an audio url
+ * 1) stop the word by word audio player if it's currently playing
+ * 2) pause the main audio player if it's currently playing
+ * 3) play the word by word audio
+ * 4) resume the main audio player it it's previously was playing
+ *
+ * Terms
+ * - main audio player refer to the audio player in the bottom navbar, this audio player plays the entire chapter
+ * - word by word audio player refer to the audio player that play the clicked word
+ *
+ * @param {string} url
+ */
+const playWordByWordAudio = (url: string) => {
```

Comment:
Nothing change in this `playWordByWordAudio` just moving the function from `QuranWord.tsx` into this file.

The new code is only `onClickQuranWord`